doc/Makefile

```diff
@@ -21,6 +21,7 @@ export PACKAGE_VERSION
 
 pgbouncer.%: pgbouncer_%.md
 	$(PANDOC) -s -t man -o $@ $<
+	rm $<
```

Comment:
I guess this is the important part of the PR for you. I think I'm fine with this, but I don't fully understand why this matters in your case. Are you running multiple builds at the same time on the same directory?

Reply:
Let me describe the situation again in more detail.

We have a build process of the product Pangolin, which includes forks of Postgress and Pgbouncer.
After assembly, various tests are launched. Our team is quite large and there are more than 500 similar build-test launches per day.
From time to time we have the problem that the Pgbouncer fork did not build. Having sorted it out, we found out that the problem is in the parallel assembly of documentation,
when one branch of the parallel assembly can delete the necessary data for another.
If you run it once, you won’t see the problem (otherwise everyone would be talking about it), but if you do it in a loop with some extra processes on server
> make
> make clean
then the problem will be revealed.

We have a parallel build process, within which various extensions are also built, and this parallelization creates an additional random load on the hardware, 
which leads to different timings when building documentation